packages/tools/webpack-fluid-loader/src/multiDocumentServiceFactory.ts

```diff
@@ -55,6 +56,7 @@ export function getDocumentServiceFactory(
         ),
         new RouterliciousDocumentServiceFactory(
             routerliciousTokenProvider,
+            undefined as any as IUrlResolver,
```

Comment:
this is super hacky, will this even continue to work? my guess is the webpack loader will blow up in the create new case for routerlicious

Reply:
Ah yes, this is in my TODOs. I'm working on refactoring this one.

Reply:
not relevant after the rollback.